playnest.sql

```diff
@@ -38,8 +39,15 @@ CREATE TABLE Comment (
     CONSTRAINT fk_user FOREIGN KEY (user_id) REFERENCES users(id),
     CONSTRAINT fk_thread FOREIGN KEY (thread_id) REFERENCES thread(id),
     CONSTRAINT fk_parent_comment FOREIGN KEY (parent_comment_id) REFERENCES comment(id)
+);-- users table: 
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    username VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    password_hash VARCHAR(255) NOT NULL,
+    registration_token VARCHAR(255)
```

Comment:
Not sure why this happened, but there's a duplicate users table here. I'll clean it up in a future commit.